spec/model/model_spec.cr

```diff
@@ -845,4 +845,40 @@ module ModelSpec
       end
     end
   end
+
+  describe "Clear::Model::JSONDeserialize" do
+    it "can create a new model from json" do
+      temporary do
+        reinit
+
+        u1_body = {first_name: "Duke"}
+        u1 = User.from_json(u1_body.to_json)
+        u1.first_name.should eq(u1_body["first_name"])
+
+        u2_body = {first_name: "Steve"}
+        u2 = User.new(u2_body.to_json)
+        u2.first_name.should eq(u2_body["first_name"])
+
+        u3_body = {first_name: "Caspian"}
+        u3 = u2.set(u3_body.to_json)
+        u3.first_name.should eq(u3_body["first_name"])
+
+        u4_body = {first_name: "George"}
+        u4 = User.create(u4_body.to_json)
+        u4.should eq(true)
```

Comment:
why is the return type a bool here?

Reply:
See line 94 https://github.com/anykeyh/clear/blob/5e233e549894010ed68ca949edf8f532d4fa086f/src/clear/model/modules/has_saving.cr

From my understanding, `save!` returns the the model and throw error if validation fails. Whereas `save` returns a boolean.

Reply:
yeap my bad, already resolved